db/schema.rb

```diff
@@ -12,4 +12,7 @@
 
 ActiveRecord::Schema.define(version: 0) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
```

Comment:
Is this line necessary?

Reply:
@tomas-stefano nope. But it should have been part of a previous PR but it wasn't included so I included it here. It's too small to leave it for another PR